lib/core/repository/cms/page_feedback.rb

```diff
@@ -2,6 +2,8 @@ module Core
   module Repository
     module CMS
       class PageFeedback < ::Core::Repository::CMS::CMS
+        MAS_CMS_API_TOKEN = ENV.fetch('MAS_CMS_API_TOKEN')
```

Comment:
The app will not be started. But I think that is the desirable outcome because the app should have the environment variable otherwise could expose a security breach on the app. So if the deployment fails you can rollback to a more secure package instead. **But will not fail because it will guaranteed by the scripts repo when it is deployed).